controllers/topics.controllers.js

```diff
@@ -123,3 +124,27 @@ exports.getAllUsers = (req, res, next) => {
       next(err);
     });
 };
+
+exports.addArticle = (req, res, next) => {
+  let { author, title, body, topic, article_img_url } = req.body;
+  if (!article_img_url) {
+    article_img_url = "https://northcoders.com/";
+  }
+  
+
+  insertArticle(author, title, body, topic, article_img_url)
+    .then((insertedArticle) => {
+      const { article_id } = insertedArticle[0];
+
+      return selectArticlesById(article_id);
+    })
+    .then((articles) => {
+      
+      res.status(201).send({ articles });
+    })
+
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
The spacing on this model is a bit weird - I recommend putting it through your formatter